### PR TITLE
Support absolute paths and namespaced classes in Presenter#t methods

### PR DESCRIPTION
**Pull request by mjhoy**
This should make the `t!` and `t` methods more useful.

### CODE REVIEW

File: spec/livery/presenter_spec.rb [hunk 1]
```diff
@@ -134,6 +134,15 @@ class SingleDummyPresenter < Struct.new(:resource); end
     it 'accepts arguments to be used in the translation' do
       expect(subject.t('.single_arg', arg: 'foobar')).to eq 'this has one argument, which is foobar'
     end
+
+    it 'allows absolute paths' do
+      expect(subject.t('other.scope')).to eq 'a different translation'
+    end
+
+    it 'works with namespaced classes' do
+      subject =  DummyNamespace::SingleDummyPresenter.new(double)
```
Comment (krmannix) [inline spec/livery/presenter_spec.rb]: 🎻 extra space here

File: lib/livery/presenter.rb [hunk 2]
```diff
@@ -89,5 +89,15 @@ def to_model
...
       @resource.to_param(*args)
     end
+
+    private
+
+    def resolve_i18n_path(key)
+      if key[0] != '.'
+        key
+      else
+        ActiveSupport::Inflector.underscore(self.class.name).tr('/', '.') + '.' + key
```
Comment (krmannix) [inline lib/livery/presenter.rb]: 🎻 I _think_ that `key` will have `.` prefixed already, based on the `if` condition (that might mean the explicit string concatenation of `.` can be removed

File: spec/livery/presenter_spec.rb (hunk 1 above)
Comment (mjhoy) [reply spec/livery/presenter_spec.rb]: 66eb05abf6ddbd9f149ffd4d68fb8a502d4ff2fe — we should consider getting rubocop set up

File: lib/livery/presenter.rb (hunk 2 above)
Comment (mjhoy) [reply lib/livery/presenter.rb]: yep. interesting, looks like extra `.` don't affect how `I18n` resolves. fc6b3852143ce9e6093a7b5b30c38c541df968a6

**Comment by mjhoy** [comment]
> minor version bump?

81e523b459b0defe82013db5516f360d9e31ab81

**Pull request merged**